src/Microsoft.PowerShell.Commands.Diagnostics/GetCounterCommand.cs

```diff
@@ -317,7 +316,7 @@ private void ProcessListSet()
         //
         private void ProcessListSetPerMachine(string machine)
         {
-            StringCollection counterSets = new StringCollection();
+            List<string> counterSets = new List<string>();
```

Comment:
Please use `var` in all LHS positions when the RHS uses an explicit type name like this

Reply:
In C#9  we can use a target-typed new expression: `List<string> counterSets = new ();`

Reply:
It is better is the same patter over all code base. So here we should use traditional `var` pattern. Later we could switch to new pattern.

Reply:
Would it not be better to make style changes in seperate PR, e.g. #13434